src/libraries/System.Net.Http/tests/TrimmingTests/System.Net.Http.TrimmingTests.proj

```diff
@@ -0,0 +1,11 @@
+<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+
+  <ItemGroup>
+    <TestConsoleAppSourceFiles Include="HttpClientTest.cs">
+      <SkipOnTestRuntimes>browser-wasm</SkipOnTestRuntimes>
```

Comment:
Doesn't disabling this test on wasm basically invalidate it? This test was trying to ensure Quic gets trimmed on non-supported platforms.

The only non-supported platform this test runs on is wasm. So I don't see the point of these tests.

Reply:
It crashes on wasm, I don't know how to fix that. But I'd love to still have this test if possible. 
I also thought we plan to widen the trimming test for platforms like Android, iOS, for which we definitely should have this test.

Reply:
The "headers" failure indicated that http-related JS APIs were being accessed inside of v8. Whether or not that indicates the trimming was successful is hard to say, but i would confidently assert that no code relying on the ability to issue http requests is going to pass inside the v8 shell, so for that test to pass it needs to have the HTTP bits *entirely* mocked or shimmed out

Reply:
>  we plan to widen the trimming test for platforms like Android, iOS, 

That's the eventual plan, yes. It currently isn't scheduled though.

Reply:
So should I remove it for now? If so, I'll do it in a follow up PR, no problem, just say so.

Reply:
It's fine to leave it. Once the other platforms come on line this will add value.